standup_notes/resources/standup-notes.bash

```diff
@@ -12,13 +12,10 @@ _standup_completions()
     days="--today --tomorrow --yesterday"
```

Comment:
I couldn't comment on the `opts=` line but i think the --editcopy and --post options need to be added to it.

Reply:
Added options to ```opts```